keps/sig-node/4033-group-driver-detection-over-cri/README.md

```diff
@@ -583,39 +310,12 @@ fixing a broken/misbehaving node if the kubelet cgroupDriver option (or
 
 ###### Are there any tests for feature enablement/disablement?
 
-<!--
-The e2e framework does not currently support enabling or disabling feature
-gates. However, unit tests in each component dealing with managing data, created
-with and without the feature, are necessary. At the very least, think about
-conversion tests if API types are being modified.
-
-Additionally, for features that are introducing a new API field, unit tests that
-are exercising the `switch` of feature gate itself (what happens if I disable a
-feature gate after having objects written with the new field) are also critical.
-You can take a look at one potential example of such test in:
-https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
--->
-
```

Comment:
presumably the line below can now reference those tests?

Reply:
The removed block was a comment block and not related to this KEP or feature.